meta-flutter/classes/embedded-linux.bbclass

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "Common parts of the recipe for flutter-embedded-linux"
+AUTHOR = "Hidenori Matsubayashi"
+HOMEPAGE = "https://github.com/sony/flutter-embedded-linux"
+BUGTRACKER = "https://github.com/sony/flutter-embedded-linux/issues"
+LICENSE = "BSD 3-Clause"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=79ca841e7b9e09b0401186f2aa334adf"
+
+SRC_URI = "git://github.com/HidenoriMatsubayashi/flutter-embedded-linux.git;protocol=https"
+SRCREV = "${AUTOREV}"
+S = "${WORKDIR}/git"
+TOOLCHAIN = "clang"
+DEPENDS += "libxkbcommon  \
+            virtual/egl \
+            flutter-engine"
+
+RDEPENDS_${PN} = "flutter-engine"
+FILES_${PN} = "${bindir}"
```

Comment:
What is that for?

Reply:
>RDEPENDS_${PN} = "flutter-engine"

Remove unnecessary definitions.

>FILES_${PN} = "${bindir}"

Without this definition, you will get an error. FILES_${PN} specifies the files to go into the main package. During packaging, the do_package task goes through PACKAGES and uses the FILES variable corresponding to each package to assign files to the package.